### DISCUSSION TITLE
Ram BottleNecking

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#4887] [Support]
Hello, I have a Dedicated Server. I have noticed that the RAM is bottlenecking. I run a Modded Minecraft Server.
![Screenshot 2023-10-10 225844](https://github.com/pterodactyl/panel/assets/93173738/4ec1211e-244b-48ac-8caa-5523b991cd25)

**Comment by Sharktheone** [comment]
There is an issue related to this (#4635). You can also find there another startup command, if you don't already are using this.

**Comment by ghost** [comment]
Okay, what is the start-up command?

**Comment by Sharktheone** [comment]
Here you can find the startup command https://github.com/pterodactyl/panel/issues/4635#issuecomment-1500829214. But check the issue for updates. Because  some people mentioned, this issue is somehow related to docker. You can also try a different JVM (HotSpot, OpenJ9, GraalVM, etc)
